projpicker/projpicker.py

```diff
@@ -2906,6 +2911,10 @@ def main():
     max_bbox = args.max
     infile = args.input
     outfile = args.output
+    webapp = args.web
+    if args.web is True:
```

Comment:
Not using `webapp`? The web script needs many arguments, so is it actually a good idea to start the web app from the core CLI? Maybe, with defaults? If we decide to include this option here, it needs to start both the server and client.